kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -25,9 +25,9 @@
       :deviceName="device['device_name']"
       :deviceIcon="getDeviceIcon(device)"
       :channels="device.channels"
-      :totalChannels="device['total_channels']"
+      :totalChannels="device['total_count']"
       :pinIcon="getPinIcon(true)"
-      :disablePinDevice="device['instance_id'] === studioId"
```

Comment:
This change is perfectly aligned with the designs, but let me check with the team at first before we merge it because disabling pinning the Kolibri Library here is looking pretty intentional so now I am not clear if the deviation from the designs had a reason or if it's just our oversight.

Reply:
Let's revert this change - the designs are out of date, and this should indeed be disabled instead.